### DISCUSSION TITLE
milvus2.2.6单机版本docker不能直接运行. 提示ERROR Unable to use the drive /minio_data: Drive /minio_data: found backend type fs, expected xl or xl-single - to migrate to a supported backend visit https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html: Invalid arguments specified

### DISCUSSION THREAD

**Discussion by zhenzi0322** [milvus-io/milvus#23510] [Ideas & Feature requests]
https://milvus.io/docs/install_standalone-docker.md

ERROR Unable to use the drive /minio_data: Drive /minio_data: found backend type fs, expected xl or xl-single - to migrate to a supported backend visit https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html: Invalid arguments specified

**Comment by xiaofan-luan** [comment]
see https://github.com/milvus-io/milvus/discussions/23171
